docs/client/README.md

```diff
@@ -32,6 +32,10 @@ After you have installed `kubectl` or `kn`, these tools will search for the `kub
 
 A `kubeconfig` file is usually automatically created when you create a Kubernetes cluster.
 
+You can also specify the configuration by setting the environment variable `$KUBECONFIG` that points to the config file.
+
+`kn` has the `--config` option which can also be used for specifying a config file like `--config path/to/config`.
+
```

Comment:
Do these two new lines refer to two additional ways you can specify a configuration? If yes, it might be easier to read if these were bullet points. For example:

> You can also specify a config file with the following options:
> - Setting the environment variable `$KUBECONFIG`, which points to the config file.
> - Using the `kn` CLI `--config` option, for example, `--config <path-to-config>`.

Reply:
Also, I'm wondering which `kn` command can you run with the `--config` option?

Reply:
It's a global option so it could work with any command. I will add an improved example.

Reply:
@vyasgun @snneji sorry I'm bit late to this one. However, I think the added content is mixing `kubeconfig` and kn's native config file that's outside of "Connect to cluster" scope.

There're 2 relevant flags provided by `kn` to the story:
```
     --context string      name of the kubeconfig context to use
     --kubeconfig string   kubectl configuration file (default: ~/.kube/config)

```

- `--kubeconfig` is equal to env variable `$KUBECONFIG`
- `--context` can be used to further specify name of context to use from existing `kubeconfig`, e.g. listed as `kubectl config get-contexts `

@vyasgun let me know if I should create a PR to correct the docs.

Reply:
I will create another one with these two options covered.